plugins/storage/redis/src/hopeit/redis_storage/__init__.py

```diff
@@ -50,13 +50,34 @@ async def get(self, key: str, *, datatype: Type[DataObject]) -> Optional[DataObj
             return Payload.from_json(payload_str, datatype)
         return None
 
-    async def store(self, key: str, value: DataObject):
+    async def store(self, key: str, value: DataObject, **kwargs):
         """
         Stores value under specified key
 
         :param key: str
         :param value: DataObject, instance of dataclass annotated with @dataobject
+        :param **kwargs: extra args
+
         """
         assert self._conn
         payload_str = str(Payload.to_json(value))
-        await self._conn.set(key, payload_str)
+        await self._conn.set(key, payload_str, **kwargs)
+
+    async def delete(self, key: str):
```

Comment:
1) Test is missing for this method
2) this new method should be added to fs_storage as well

Reply:
added